bucket/meson.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.61.1",
+    "description": "Open source build system meant to be both extremely fast and user friendly.",
+    "homepage": "http://mesonbuild.com",
+    "license": "Apache-2.0",
+    "suggest": {
+        "xpdf-tools": "ninjia"
+    },
```

Comment:
```suggestion
    "suggest": {
        "Ninja": "ninja",
        "Python": "python"
    },
```

Reply:
copied the section from another manifest and forget to edit it. 😂